WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -21,6 +21,11 @@ class ReaderTabView: UIView {
     private let viewModel: ReaderTabViewModel
 
     private var filteredTabs: [(index: Int, topic: ReaderAbstractTopic)] = []
+    private var previouslySelectedIndex: Int = 0
+
+    private var discoverIndex: Int? {
+        return tabBar.items.firstIndex(where: { $0.title == NSLocalizedString("Discover", comment: "Discover tab name") })
```

Comment:
The string comparison is something we were doing before but it worries me a little.  Translations could differ between the web and the apps.  I don't really have a better suggestion tho.

Reply:
I just copied it from the original implementation. It made me uncomfortable as well, but it seems to work. Since this is a patch, I didn't want to make _too_ many changes. I've added it to my list to look at later.